compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/window/Popup.skiko.kt

```diff
@@ -17,11 +17,38 @@
 package androidx.compose.ui.window
 
 import androidx.compose.runtime.Composable
+import androidx.compose.runtime.Immutable
 import androidx.compose.runtime.remember
 import androidx.compose.ui.Alignment
-import androidx.compose.ui.input.key.KeyEvent
+import androidx.compose.ui.ExperimentalComposeUiApi
+import androidx.compose.ui.input.key.*
 import androidx.compose.ui.unit.IntOffset
 
+@Immutable
+actual class PopupProperties actual constructor(
+    actual val focusable: Boolean,
```

Comment:
May be we should deprecate 2 functions `Popup(focusable = )` in favor of `Popup(properties = PopupProperties(focusable = ))`. What do you think?

Reply:
I'd keep it for now because:
1. It's stable already
2. it allows to handle keyboard input (new overload doesn't)

Reply:
Ok, please make an issue. This is design incosistent, we shouldn't forget about it.

Reply:
- https://github.com/JetBrains/compose-multiplatform/issues/3350